ci/README.md

```diff
@@ -0,0 +1,3 @@
+# Continuous Integration Tests
+
+This folder contains pipeline configurations and any ancillary files needed for our Continuous Integration test pipelines.
```

Comment:
Let's follow this description up with details about each pipeline? Maybe copying from that Confluence doc? Not necessary here but might be helpful for future devs